topology/doc/user_manual_scripts/connections.py

```diff
@@ -24,6 +24,7 @@
 import nest
 import nest.topology as tp
 import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d.axes3d import Axes3D
```

Comment:
This is required to enable [3d projection here](https://github.com/heplesser/nest-simulator/blob/63fc287e0ba00d1c02c13bca4a4c8cbeea1c1352/topology/doc/user_manual_scripts/connections.py#L223). I am not sure why it was not needed before. That may be due to internal changes in matplotlib.